front/src/Phaser/Game/GameManager.ts

```diff
@@ -0,0 +1,31 @@
+import {GameSceneInterface, GameScene} from "./GameScene";
+import {ROOM} from "../../Enum/EnvironmentVariable"
+import {Connexion} from "../../Connexion";
+
+export let ConnexionInstance : Connexion;
+
+export interface GameManagerInterface {
+    GameScenes: Array<GameSceneInterface>;
+
+    sharedUserPosition(UserPositions: any): void;
+}
+export class GameManager implements GameManagerInterface {
+    GameScenes: Array<GameSceneInterface> = [];
```

Comment:
I don't understand why we need an array of GameScene here.
At a given time, we can only be in one room at the time (there is no way we are playing in 2 rooms at the same time).
So this class does not make much sense to me.

Reply:
Yes, we cannot be in two scenes at the time. But if we want more skin for the application, we must switch between the scenes and at this moment have more scenes initalisated.
I think that in this application, we can imagine having more room and more scene.  (We can manage more scene in this same map with more terrain, but in the back we lose the room feature)
What did you think?